randovania/games/samus_returns/logic_database/Area 2 - Entrance.txt

```diff
@@ -271,7 +275,7 @@ Extra - asset_id: collision_camera_005
           Lightning Armor or Use Spider Ball
           Movement (Intermediate) and Normal Damage ≥ 99
   > Door to Transport to Area 2 Exterior (Top)
-      Trivial
+      After Area 2 (Entrance) - Transport to Area 2 Exterior Unlock Door or Enabled Door Lock Randomizer
```

Comment:
Like I said in the discord, I don't think that is the right way to do it. An enabled door lock randomizer doesn't force that this door is changed.
I'm not sure if this counts for individual door locks but for sure it does for type door lock rando. (where I guess it's never changed because of one sided door?)
In that case you still have to collect the event even with door lock randomizer enabled.

Reply:
Yeah this is fine for regular door lock by brekmaks for types. Maybe there's a way to specify which mode is used? Or we just check for door lock and if true, change both sides to be power regardless